packages/react-components/src/components/styles/VoiceOverButton.style.ts

```diff
@@ -23,5 +24,6 @@ export const buttonStyle = mergeStyles({
  */
 export const iconWrapperStyle = mergeStyles({
   position: 'absolute',
-  pointerEvents: 'none'
+  pointerEvents: 'none',
+  bottom: '2.6rem'
```

Comment:
Need to figure out why there is 2.6rem difference in drawing the icon

Reply:
Solve the problem by removing the VoiceOverButton and apply the `pointerEvents: 'none', to `onRenderIcon` props in TextField, thanks to @mgamis-msft solution